README.md

```diff
@@ -1,53 +1,34 @@
 Motor Board v4 Firmware
 =======================
 
-Prerequisites
--------------
+## Instructions
 
-These instructions assume that you are using a version of Linux, but should also work on BSD-derived OSes (including OS X). Windows users may wish to try [Cygwin][cygwin], create a Linux VM (although flashing the board from a VM may require some thought), or consider their life choices.
+Using a posix system, you require `make`, the `arm-none-eabi` toolchain and `git`.
+Before attempting to build anything initialise all the submodules.
+```shell
+$ git submodule update --init --recursive
+```
 
-* `git` can be found in your distribution's repositories, or [online][git].
-* A working C compiler, such as `gcc` or `clang`, also from your distribution's repositories.
-* `make`, also from your distribution's repositories.
-* The `arm-none-eabi` toolchain. This may be in your distribution's repositories, otherwise download it [from Launchpad][toolchain] and follow the instructions in the README.
-* `stm32flash`, which can be found [on Google Code][stm32flash]. After downloading, build and install it:
+To build the main binary, run:
+```shell
+$ make
+```
+The binary will then be at `src/main.bin`.
+This will also build the library libopencm3 the first time you run it.
 
-	$ make
-	$ sudo make install
+This can be flashed to an attached motor board that is in bootloader using:
+```shell
+$ make -C src prog
+```
+To use the `prog` command you need to install stm32flash. This is a cross-platform utility that may be in your operating system's package manager, otherwise it can be downloaded from [their website](https://sourceforge.net/p/stm32flash/wiki/Home/).
 
-Instructions
-------------
+To enter the bootloader the pushbutton on the board can be pressed with the 12V input connected. Whilst 12V power is present the board will remain in bootloader.
+Alternatively the bootloader can be entered using the corresponding command.
```

Comment:
nit: We should consider specifying the command here if we're going to refer to it. Especially as it differs between the old and new firmware